systemvm/debian/etc/systemd/system/cloud.service

```diff
@@ -9,5 +9,4 @@ WantedBy=multi-user.target
 Type=simple
 WorkingDirectory=/usr/local/cloud/systemvm
 ExecStart=/usr/local/cloud/systemvm/_run.sh
-Restart=always
-RestartSec=5
+Restart=on-failure
```

Comment:
Setting it to *always* was just fine, why this particular change?

Reply:
@wido looks like in my test case I hit - https://unix.stackexchange.com/questions/289629/systemd-restart-always-is-not-honored/324297#324297